Makefile

```diff
@@ -414,13 +421,28 @@ verify-packages:
 
 .PHONY: verify-gendocs
 verify-gendocs: kops
-	KOPS_STATE_STORE= hack/verify-gendocs.sh
+	@: FIXME: why unset this?; \
+	export KOPS_STATE_STORE=; \
```

Comment:
I do not think we need to anymore. You can test by removing and setting the car in your shell and seeing if running the command will embed the state store in the docs. I think this issue was already addressed in another pr.